src/interfaces/store-interfaces/state-interfaces/tombstone-interface.ts

```diff
@@ -20,4 +20,5 @@ export interface TombStoneIF {
   haveUnsavedChanges: boolean
   folioNumber: string
   transactionalFolioNumber: string
+  correctionType: CorrectionErrorTypes
```

Comment:
Is this the right place to place enum `CorrectionErrorTypes`?

Reply:
Yes, this property should be of this type.

No, because I'm moving this property into a new store correction object.

I'd really like to get my refactor changes/fixes in before you guys do too much more in this UI. It will take me a day. Do you have things that must go in ASAP? Do you have other work you can do while waiting?